chatbot-langchain/ai-studio.yaml

```diff
@@ -15,11 +15,13 @@ application:
         - amd64
       ports:
         - 8001
+      image: quay.io/redhat-et/llamacpp-server
     - name: streamlit-chat-app
       contextdir: .
       containerfile: builds/Containerfile
       arch:
         - arm64
         - amd64
       ports:
-        - 8501
\ No newline at end of file
+        - 8501
+      image: quay.io/redhat-et/streamlit-chat-app
```

Comment:
This one should be `quay.io/redhat-et/locallm-chatbot:latest` (we can rename, but that's what it is atm)

Reply:
It's the same for us, so what's good for you, it's good for us 👍